### DISCUSSION TITLE
What backend storage can i use?

### DISCUSSION THREAD

**Discussion by AndreyBulezyuk** [ether/etherpad-lite#5332] [Q&A]
Hi, generally speaking, what backend storage can use for my documents?

Can I use 

- local storage
- gcp storage
- aws s3
- ms onedrive
- google drive
- ... ?

Is any data being sent to etherpad server such metadata, contents, user activity?

**Comment by rhansen** [comment]
Etherpad uses [ueberdb](https://github.com/ether/ueberDB#readme) for storage, so any database supported by that will work (e.g., MySQL, PostgreSQL, SQLite).
